packages/react-scripts/template/README.md

```diff
@@ -858,6 +859,44 @@ Snapshot testing is a new feature of Jest that automatically generates text snap
 
 This feature is experimental and still [has major usage issues](https://github.com/facebookincubator/create-react-app/issues/372) so we only encourage you to use it if you like experimental technology. We intend to gradually improve it over time and eventually offer it as the default solution for testing React components, but this will take time. [Read more about snapshot testing.](http://facebook.github.io/jest/blog/2016/07/27/jest-14.html)
 
+## Turbo Charge Component Development with React Storybook
+
+Usually, in our app, we have a lot of UI components, and they've different states. 
+For an example, a simple button component could have following states:
+
+* With a text label
+* With some emoji
+* With disabled mode
+
+With React Storybook, you can manage these different states (or stories) and develop them without running your app. You can also deploy your Storybook as a static app.
+
+That means, everyone in your team can view and review different states of UI components without starting a backend server or creating an account in your app.
+
+![React Storybook Demo](http://i.imgur.com/7CIAWpB.gif)
+
+**Here's how to setup your app with Storybook:**
+
+First, install the following NPM module globally:
+
+```sh
+npm install -g getstorybook
+```
+
+Then, apply following command inside your app:
+
+```sh
+getstorybook
```

Comment:
Hey, I finished all of them and we are ready to go.
See: https://github.com/kadirahq/react-storybook/issues/468